### DISCUSSION TITLE
[Feature] Jump to specific date in the timeline view

### DISCUSSION THREAD

**Discussion by lefuturiste** [immich-app/immich#3873] [Feature Request]
### The feature

If I want to see the picture of a specific date in the main, AFAIK there is no other ways to do it than manually selecting the approximate date with the month selector on the right or scrolling down the timeline.

The goal of this feature would be to allow a user to jump to a specific date they ask on the timeline (via a date picker or via a form with ISO-8601 date format eg. 2021-08-26).
Jumping to the right place would also allow the user to look around (days past and ahead) to have some context.

I'm interested working on this feature on the frontend.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4115
